openshift/oadp-operator
# [oadp-1.4] OADP-4279 CVE-2023-45288, update x/net

## Why the changes were made

Proactive go mod update for x/net - CVE-2023-45288

## How to test the changes made

Double check me please :)

10056  git checkout -b oadp-1.4 origin/oadp-1.4
10057  go mod tidy
10058  go get golang.org/x/net
10059  git status
10060  go mod tidy
